pkg/grpc/authn/userpki/extractor.go

```diff
@@ -73,7 +73,7 @@ func (i extractor) IdentityForRequest(ctx context.Context, ri requestinfo.Reques
 			}
 			resolvedRoles, err := provider.RoleMapper().FromUserDescriptor(ctx, ud)
 			if err != nil {
-				logging.GetRateLimitedLogger().WarnL(
+				logging.GetRateLimitedLogger().DebugL(
```

Comment:
I'm actually not sure why we need these in addition to the logging in authn interceptor. Blaming suggests it is related to log rate limiting, so some code archeology is necessary.